integration-tests/contract-estimation-tests.spec.ts

```diff
@@ -39,8 +39,8 @@ CONFIGS().forEach(({ lib, setup, knownBaker, createAddress, protocol, rpc }) =>
       }
     })
 
-    florencenet('Estimate transfer with allocated destination', async (done) => {
-      const estimate = await LowAmountTez.estimate.transfer({ to: await Tezos.signer.publicKeyHash(), amount: 1.9 });
+     florencenet('Estimate transfer with allocated destination', async (done) => {
+      const estimate = await LowAmountTez.estimate.transfer({ to: await Tezos.signer.publicKeyHash(), amount: 1.9 }, { includeRevealOperation: false });
```

Comment:
It looks like defaulting to false instead minimize the impact of this change. What are the reason we preferred defaulting to true?

Reply:
I switched this back to false by default as it makes more sense in my opinion. 

When a user specifies his own fees when doing an operation, the fees should not include fees related to the reveal operation as Taquito will take care of adding them if needed. If we returned an estimate that contains the reveal fee and the user uses this when sending an operation, it will be ending up adding two times the fees for the reveal operation.